### PR TITLE
Add info.cemu.Cemu

### PR DESCRIPTION
**Pull request by ColinKinloch**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by orowith2os** [comment]
bot, build info.cemu.Cemu

**Comment by ColinKinloch** [comment]
bot, build info.cemu.Cemu

**Comment by ColinKinloch** [comment]
bot, build info.cemu.Cemu

**Comment by ColinKinloch** [comment]
bot, build info.cemu.Cemu

**Comment by hfiguiere** [comment]
PR should be done against the `new-pr` branch

File: info.cemu.Cemu.yml [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+  - name: Cemu
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DPORTABLE=false
+      - -DENABLE_VCPKG=false
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_C_COMPILER=/usr/bin/gcc
+      - -DCMAKE_CXX_COMPILER=/usr/bin/g++
+      - -G Ninja
```
Comment (hfiguiere) [inline info.cemu.Cemu.yml:176]:
```suggestion
```

This is what `cmake-ninja` does

File: info.cemu.Cemu.yml [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+          tag-template: v$version
+
+  - name: Cemu
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DPORTABLE=false
+      - -DENABLE_VCPKG=false
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_C_COMPILER=/usr/bin/gcc
+      - -DCMAKE_CXX_COMPILER=/usr/bin/g++
```
Comment (hfiguiere) [inline info.cemu.Cemu.yml:175]: aren't these set already?

File: info.cemu.Cemu.yml [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+      - -G Ninja
+      - -DCMAKE_MAKE_PROGRAM=/usr/bin/ninja
```
Comment (hfiguiere) [inline info.cemu.Cemu.yml:177]: isn't this set too ?

File: info.cemu.Cemu.yml [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+          type: anitya
+          project-id: 6845
+          stable-only: true
+          url-template: https://boostorg.jfrog.io/artifactory/main/release/$version/source/boost_${major}_${minor}_$patch.tar.bz2
+
+  - name: libzip
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_INSTALL_LIBDIR=/app/lib
+      - -DCMAKE_INSTALL_INCLUDEDIR=/app/include
```
Comment (hfiguiere) [inline info.cemu.Cemu.yml:55]: I believe all of this is set automatically

**Comment by ColinKinloch** [comment]
I can't seem to change the target branch new MR  #3975

**Pull request closed without merging**